### PR TITLE
Added support for language info retrival

### PR DESCRIPTION
**Pull request by viviivanov**

### CODE REVIEW

File: src/lib/connection/connector/fallback-handlers.js [hunk 1]
```diff
@@ -67,3 +67,14 @@ export function isIos() {
 export function openLink(url) {
   window.open(url, '_blank');
 }
+
+/**
+ * Open an external link
+ *
+ * @return {Object} containing various language informations
+ */
```
Comment (staust) [inline src/lib/connection/connector/fallback-handlers.js]: wrong/c&p documentation

File: src/lib/connection/connector/fallback-handlers.js [hunk 2]
```diff
@@ -67,3 +67,14 @@ export function isIos() {
...
+export function langInfos() {
+  return {
+    contentLanguages: ['en_EN', 'de_DE']
```
Comment (staust) [inline src/lib/connection/connector/fallback-handlers.js]: Is this list complete?

Comment (viviivanov) [reply src/lib/connection/connector/fallback-handlers.js]: no will be a followup

File: src/lib/connection/connector/fallback-handlers.js (hunk 1 above)
Comment (viviivanov) [reply src/lib/connection/connector/fallback-handlers.js]: - [ ] fix

**Pull request merged**